actions/websocket_actions.jsx

```diff
@@ -182,9 +182,11 @@ export function unregisterPluginReconnectHandler(pluginId) {
     Reflect.deleteProperty(pluginReconnectHandlers, pluginId);
 }
 
-export function reconnect(includeWebSocket = true) {
+export function reconnect(includeWebSocket = true, init = false) {
```

Comment:
Also, this is a bit of nitpicking, but perhaps we could call the argument `reinit`? For some reason, it took me a minute to realize what `init` meant in this context